global-script.lua

```diff
@@ -1859,20 +1859,39 @@ function timePassesCo()
     timePassing = false
     return 1
 end
+function getReminderColor(name)
+    return string.match(name, "^(%a*)'s %a* Reminder Tokens")
+end
 function handlePiece(object, depth)
-    if string.sub(object.getName(),1,4) == "City" then
+    local isoOffset = Vector(-11.2, 2, -1)
+    local defOffset = Vector(-9.5, 2, -1)
+    local name = object.getName()
+    local spos = getObjectFromGUID(elementScanZones["Red"]).getPosition()
```

Comment:
what's the point of declaring the local variable here if you never use this value?

Reply:
It is used in a later commit. I'll move it there.